### DISCUSSION TITLE
Does Trivy support scanning python zipapps?

### DISCUSSION THREAD

**Discussion by BeyondEvil** [aquasecurity/trivy#4239] [Q&A] [answered]
Like the title says, does Trivy support scanning python zipapps (.pyz extension).

I have a docker image, based on Chainguard/Wolfi with a zipapp built by [Shiv](https://github.com/linkedin/shiv). And it reports zero vulnerabilities. Which I suspect is highly unlikely.

**Comment by knqyf263** [comment, marked as answer]
I don't think it is supported.
https://aquasecurity.github.io/trivy/v0.41/docs/scanner/vulnerability/language/python/

**Reply by BeyondEvil** [reply]
Yeah, I figured as much.

They're basically zip-files, which I _do_ think Trivy supports.

Hence, adding support should be trivial ™️ 

I'm happy to contribute a PR, but I would need some initial guidance.

**Reply by knqyf263** [reply]
I've opened an issue. We need help from someone familiar with Python zipapps.
https://github.com/aquasecurity/trivy/issues/4240
